### PR TITLE
[CMake] Update CMake script for /share and /etc.

### PR DESCRIPTION
**Pull request by AYESDIE**
Fixes: #1225

### CODE REVIEW

**Comment by ice0** [comment]
@AYESDIE you need to copy them on build stage, not on install stage. 

The main idea is to allow user just open project in CLion (for example) and just build it and run. 
As for now it requires to do `make install` and choose another binary (not produced by `make` command)

**Comment by AYESDIE** [comment]
The building process is still hectic. But I think once this is done, we will be able to run `synfig` for `build_images` target from our `cmake-build` directly which will make things easier :)

**Comment by AYESDIE** [comment]
Oh, wait, I messed it abit. I'll push update soon.

**Comment by ice0** [comment]
2 additions:
1. Try to not require CMake higher than 3.0. Synfig is distributed across multiple platforms, so it’s better not to require a higher version of CMake than you really need. (Not all of them have required version)
2. https://cmake.org/cmake/help/v3.0/command/file.html

> WRITE will write a message into a file called ‘filename’. It overwrites the file if it already exists, and creates the file if it does not exist. (If the file is a build input, use configure_file to update the file only when its content changes.)

So better to use `configure_file` for that.

**Comment by AYESDIE** [comment]
@ice0 This has been bugging me a bit. Would we not require those *.glade and modules.cfg file if we decide to `make install`? If we don't go for `install` and just copy those, they won't be installed to their appropriate location when we do `make install`. 

This seems to be a case specific thing where we want to run it on something like CLion.

**Comment by ice0** [comment]
They will be installed if you leave `install` command as is.

For example you just need to do something like:
```
foreach (UIFILE IN ITEMS ${UIFILES})
   # copying file to SYNFIG_BUILD_ROOT too
   configure_file(...)

    install(
        FILES ${CMAKE_CURRENT_SOURCE_DIR}/${UIFILE}
        DESTINATION share/synfig/ui
    )
endforeach()
```

**Comment by AYESDIE** [comment]
Thanks, and I think this should do it. 

I checked the `~/synfig-cmake` and `cmake-build/output` directory and it works.

File: synfig-core/src/modules/CMakeLists.txt [hunk 1]
```diff
@@ -73,6 +73,11 @@ endforeach(MOD)
 
 file(WRITE synfig_modules.cfg ${SYNFIG_MODULES_CONTENT})
 
+configure_file(
```
Comment (ice0) [inline synfig-core/src/modules/CMakeLists.txt]: Please add COPYONLY, as it is faster and does not require parsing the file.

**Comment by ice0** [comment]
In addition, after these changes, can you check if you can just build and run the synfigstudio target in CLion.

**Comment by AYESDIE** [comment]
Opening it requires changing configuration from CLion itself. On specifying `SYNFIG_ROOT` the application opened.

**Comment by ice0** [comment]
Awesome!

Can you take https://github.com/synfig/synfig/issues/1230?

**Comment by AYESDIE** [comment]
Sure :)

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**